packages/tools/atf/package.mk

```diff
@@ -16,7 +16,9 @@ PKG_TOOLCHAIN="manual"
 
 make_target() {
   if [ "${DEVICE}" = "iMX8" ]; then
-    CROSS_COMPILE="${TARGET_KERNEL_PREFIX}" LDFLAGS="" CFLAGS="--param=min-pagesize=0" make PLAT=${ATF_PLATFORM} bl31
+    CROSS_COMPILE="${TARGET_KERNEL_PREFIX}" LDFLAGS="--no-warn-rwx-segments" CFLAGS="--param=min-pagesize=0" make PLAT=${ATF_PLATFORM} bl31
+  elif [ "${DEVICE}" = "A64" -o "${DEVICE}" = "H5" -o "${DEVICE}" = "H6" ]; then
+    CROSS_COMPILE="${TARGET_KERNEL_PREFIX}" LDFLAGS="--no-warn-rwx-segments" CFLAGS="" make PLAT=${ATF_PLATFORM} bl31
   else
     CROSS_COMPILE="${TARGET_KERNEL_PREFIX}" LDFLAGS="" CFLAGS="" make PLAT=${ATF_PLATFORM} bl31
```

Comment:
Do we ever get here? Why don't we just use single command with all warnings disabled? I don't think there's any harm in doing that.

Reply:
Hi @jernejsk  - fixed (as it is only the iMX8 and the 64bit Allwinners that use the atf.) I would like to keep the `CFLAGS="--param=min-pagesize=0"` there - as I reminder to go back and fix this once it is fixed in upstream.